src/examples/test.yml

```
@@ -0,0 +1,22 @@
+description: >
+   Easily install and configure the gmsaas CLI automatically in your jobs or commands.
+   Start Android instance, test your app and stop instance on Genymotion Cloud SaaS platform
+usage:
+  version: 2.1
+
+  orbs:
+    genymotion-saas: genymotion/genymotion-saas@dev:alpha
```

```suggestion
    genymotion-saas: genymotion/genymotion-saas@x.y
```
We could put the name of release version, instead of x.y